docs/pages/project/contributing.md

```diff
@@ -3,13 +3,26 @@ layout: page
 title: Contributing
 permalink: project/contributing
 ---
-
 # <a name="contributing">Contributing Overview</a>
-Please do! Thanks for your help improving the project! :balloon:
 
-All contributors are welcome. Please see the [newcomers welcome guide](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit) for how, where and why to contribute. This project is community-built and welcomes collaboration. Contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
+Please do! Thanks for your help! :balloon:
+
+This project is community-built and welcomes collaboration. Contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) 
+code of conduct. 
+
+Not sure where to start? 
+
+Follow these steps and you'll be right at home.
+
+1. See the [*Newcomers Welcome Guide*](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit) for how, where and why to contribute.
 
-Not sure where to start? First, see the [newcomers welcome guide](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit). Grab an open issue with the [help-wanted label](../../labels/help%20wanted) and jump in. Join the [Slack account](http://slack.layer5.io) and engage in conversation. Create a [new issue](/../../issues/new/choose) if needed.  All [pull requests](/../../pulls) should reference an open [issue](/../../issues). Include keywords in your pull request descriptions, as well as commit messages, to [automatically close issues in GitHub](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords).
+2. Sign up for a [*MeshMate*](https://layer5.io/community#meshmate) to find the perfect Mentor to help you explore the Layer5 projects and find your place in the community:
+  - **Familiarize** yourself with all the live Layer5 projects: Spend time understanding each of the Layer5 initiatives through high level overviews available in the community drive and in discussions with your MeshMate.
```

Comment:
You'll want to delete the word "live" here. Two links that would fit well in this bullet are:

1. a link to the [community drive](https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA)
1. a link to the [Layer5 Repository Overview](https://docs.google.com/document/d/1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk/edit#heading=h.uwtb5xf7b5hw)